accord-core/src/main/java/accord/impl/InMemoryCommandStore.java

```diff
@@ -333,38 +344,6 @@ private void forEach(Routable keyOrRange, Ranges slice, Consumer<RoutableKey> fo
         }
     }
 
-    @Override
-    protected void registerHistoricalTransactions(Deps deps)
```

Comment:
moved this to safe command store... we can't handle this in `CommandStore` for C* unless we lazy load the dependencies (which requires https://github.com/apache/cassandra-accord/pull/27)